package.json

```diff
@@ -15,6 +15,7 @@
       "@clr/core/lit-html",
       "@clr/core/@storybook/**",
       "@clr/core/normalize.css",
+      "**/@clr/icons",
```

Comment:
Why this?

Reply:
This was for forcing yarn to install `@clr/icons` into apps now is no longer needed - is part of my next commit anyway